unstructured/ingest/doc_processor/generalized.py

```diff
@@ -17,15 +17,15 @@ def initialize():
     MODEL_TYPES[None]["config_path"]
 
 
-def process_document(doc: "IngestDoc") -> Optional[List[Dict[str, Any]]]:
-    """Process any IngestDoc-like class of document with Unstructured's auto partition logic."""
+def process_document(doc: "IngestDoc", **kwargs) -> Optional[List[Dict[str, Any]]]:
```

Comment:
let's rename to **partition_kwargs, and document in the doc_string something along the lines of "ultimately the params passed to `partition()`".

same goes for the other functions where the **kwargs has been introduced.